platform/darwin/src/MGLLight.h

```diff
@@ -0,0 +1,65 @@
+#import <CoreGraphics/CoreGraphics.h>
+
+#import "MGLFoundation.h"
+#import "MGLStyleValue.h"
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, MGLLightAnchor) {
+    MGLLightAnchorTypeMap,
+    MGLLightAnchorTypeViewport
+};
+
+MGL_EXPORT
+@interface MGLLight : NSObject
+
++ (instancetype)defaultLight;
+
+/**
+ `lightAnchorType` property.
+ */
+@property (nonatomic) MGLLightAnchor lightAnchor;
+
+/**
+ Values describing animated transitions to `lightAnchorType` property.
+ */
+@property (nonatomic) MGLTransition lightAnchorTransition;
```

Comment:
That's what I thought as well but looking at [light](https://github.com/mapbox/mapbox-gl-native/blob/fabian-light-property/src/mbgl/style/light.cpp#L29) it has a transition for anchor and to keep consistency I kept the property.